larcoreobj/SummaryData/GeometryConfigurationInfo.cxx

```diff
@@ -0,0 +1,55 @@
+/**
+ * @file   larcoreobj/SummaryData/GeometryConfigurationInfo.cxx
+ * @brief  Description of the current configuration of detector geometry.
+ * @author Gianluca Petrillo (petrillo@slac.stanford.edu)
+ * @date   May 13, 2020
+ * @see    larcoreobj/SummaryData/GeometryConfigurationInfo.h
+ * 
+ * This is a header-only library.
+ */
+
+#include "larcoreobj/SummaryData/GeometryConfigurationInfo.h"
+
+// C/C++ standard library
+#include <ostream>
+
+
+// -----------------------------------------------------------------------------
+std::ostream& sumdata::operator<<
+  (std::ostream& out, sumdata::GeometryConfigurationInfo const& info)
+{
+  
+  if (info.isDataValid()) {
+    out <<   "Geometry information version: " << info.dataVersion;
```

Comment:
To avoid extended nesting,  you might consider:

```.cpp
if (not info.isDataValid()) {
  return os << "Invalid geometry configuration information" << std::endl;
}

if (info.dataVersion  >= sumdata::GeometryConfigurationInfo::DataVersion_t{1}) ...
```

Reply:
Hmm... ok. Whatever reason I may have had not to do it, it was weak enough that I don't remember it any more.